redbot/cogs/audio/core/commands/controller.py

```diff
@@ -617,8 +617,13 @@ async def command_stop(self, ctx: commands.Context):
     @commands.guild_only()
     @commands.cooldown(1, 15, commands.BucketType.guild)
     @commands.bot_has_permissions(embed_links=True)
-    async def command_summon(self, ctx: commands.Context):
-        """Summon the bot to a voice channel."""
+    async def command_summon(
+        self, ctx: commands.Context, channel: Optional[discord.VoiceChannel] = None
+    ):
+        """Summon the bot to a voice channel.
+        You can specify a personalised voice room with an id. Otherwise, the bot will join the voice room you are in.
+        To learn how to get a channel ID please read the following help article <https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID>
+        """
```

Comment:
```suggestion
        """Summon the bot to a voice channel.
        
        You can specify a personalised voice room with an id. Otherwise, the bot will join the voice room you are in.
        To learn how to get a channel ID please read the following help article <https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID>
        """
```

The extra empty line here is important to break up the help into its correct sections.

Once this is done this should be good to go.